mappings/net/minecraft/class_5464.mapping

```diff
@@ -0,0 +1,86 @@
+CLASS net/minecraft/class_5464
```

Comment:
Could you attempt to name this?

Reply:
I proposed "TerrainFeatures" because it seems to contain terrain/foliage elements and no structures